tools/clang_tidy/lib/clang_tidy.dart

```diff
@@ -292,6 +299,10 @@ class ClangTidy {
     List<Command> commands,
     Options options,
   ) async {
+    io.File? configPath = options.configPath;
+    if (configPath != null && options.excludeSlowChecks) {
+      configPath = _createClangTidyConfigExcludingSlowLints(configPath);
```

Comment:
The temp directory should probably be deleted sometime after `_runJobs` finishes up on line 171.